### PR TITLE
WIP: 2018 Toyota Camry Hybrid

### PR DESCRIPTION
**Pull request by vanillagorillaa**
Toyota Camry Hybrid 2018 Port, Just needs to be tested

### CODE REVIEW

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -12,6 +12,7 @@ class ECU:
 
                (0x292, ECU.APGS, (CAR.PRIUS, CAR.RAV4H, CAR.LEXUS_RXH, CAR.RAV4, CAR.COROLLA, CAR.CAMRYH), 0,   3, '\x00\x00\x00\x00\x00\x00\x00\x9e'),
                (0x283, ECU.DSU, (CAR.PRIUS, CAR.RAV4H, CAR.LEXUS_RXH, CAR.RAV4, CAR.COROLLA, CAR.CAMRYH), 0,   3, '\x00\x00\x00\x00\x00\x00\x8c'),
+               (0x2E4, ECU.CAM, (CAR.CAMRYH), 0, 3, '\xd4\x00\x00\x00\xbf'),
```
Comment (energee) [inline selfdrive/car/toyota/values.py]: This one is the steering message

**Comment by geohot** [comment]
Does this work? Can we merge?

**Comment by vanillagorillaa** [comment]
Lateral only, stock ACC, will need to add panda FW changes first.

**Comment by vanillagorillaa** [comment]
Added panda forwarding. Uses a 120 ohm resistor on pins 12 & 13. Going to close the port for now since only lateral.

**Pull request closed without merging**